src/config/firmware.md

```diff
@@ -21,7 +21,24 @@ Install the AMD package, `linux-firmware-amd`, which contains microcode for both
 AMD CPUs and GPUs. AMD CPUs and GPUs will automatically load the microcode, no
 further configuration required.
 
-## Verification
+### Verification
 
 The `/proc/cpuinfo` file has some information under `microcode` that can be used
 to verify the microcode update.
+
+## Adding firmware
+
+A list of firmware available in the Void repositories can be found by running:
+
+```
+$ xbps-query -Rs firmware | awk '{ print $2 }' | grep firmware
```

Comment:
I think `xbps-query -p pkgver -Rs firmware | awk '{ print $2 }'` is a bit more explicit, what do you think?

Reply:
And this sort of trick could perhaps be mentioned in XBPS pages too.

Reply:
That's definitely better; i was using this because `xbps-query -Rs firmware -p pkgname` didn't work. What am i misunderstanding?

Reply:
I'm not sure eithter... I think some of the properties can't be queried for.

I couldn't query: pkgname, alternatives, automatic-install, install-date, install-script, metafile-sha256, repository and state. There's probably some difference in how these properties are generated/read from the packages.